src/OpenTelemetry/Internal/SelfDiagnostics.cs

```diff
@@ -39,6 +41,9 @@ static SelfDiagnostics()
 
         private SelfDiagnostics()
         {
+            this.eventListener = new SelfDiagnosticsEventListener(
+                EventLevel.Error,
```

Comment:
Why hard coded .Error level?

Reply:
Fixed it. I'll create EventListener in ConfigRefresher, where the configured EventLevel will be available.